reader-sdk-react-native-quickstart/android/app/src/main/java/com/rnreadersdksample/MainApplication.kt

```diff
@@ -18,10 +18,11 @@ import com.facebook.react.ReactNativeHost
 import com.facebook.react.ReactPackage
 import com.facebook.react.PackageList
 import com.facebook.soloader.SoLoader
-import com.squareup.sdk.reader.ReaderSdk
 import com.rnreadersdksample.MainApplication
+
 import android.content.Context
 import com.facebook.react.ReactInstanceManager
+import com.squareup.sdk.reader.ReaderSdk
```

Comment:
If we're alphabetizing, cool, but lots of others aren't (especially the android.* on line 23, but also rnreadersdksample and soloader are out of order).

If we _aren't_ alphabetizing, why bother shuffling this?  It just adds churn.